DashWallet/Sources/UI/CrowdNode/Portal/CrowdNodePortalViewController.swift

```diff
@@ -96,8 +101,15 @@ extension CrowdNodePortalController {
     private func configureNavBar() {
         let appearance = UINavigationBarAppearance()
         appearance.configureWithTransparentBackground()
-        let image = UIImage(systemName: "chevron.backward")?.withTintColor(.white, renderingMode: .alwaysOriginal)
-        appearance.setBackIndicatorImage(image, transitionMaskImage: image)
+        
+        let backButton = UIButton(type: .custom)
+        backButton.frame = .init(x: 0, y: 0, width: 30, height: 30)
+        backButton.setImage(UIImage(systemName: "arrow.backward"), for: .normal)
+        backButton.tintColor = .white
+        backButton.imageEdgeInsets = .init(top: 0, left: -10, bottom: 0, right: 0)
+        backButton.addTarget(self, action: #selector(backButtonAction), for: .touchUpInside)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: backButton)
```

Comment:
Seems like the only way to fix the back button and make it white is to recreate it here.

Reply:
We can also change the `tintColor` of the 'original' back button:

```
navigationItem.leftBarButtonItem!.customView!.tintColor = .white
```

This solution is better, but there is a problem that we can only do this at `viewDidAppear` which doesn't look good. Let's keep your implementation for now and I will improve `BaseNavigationController` and add extra property into `NavigationBarDisplayable` protocol, like `backButtonTintColor` and we will be able to change the back button color per view controller without adding a custom button.